rubocop.yml

```diff
@@ -129,5 +133,5 @@ Style/RescueStandardError:
 Style/TernaryParentheses:
   EnforcedStyle: 'require_parentheses_when_complex'
 
-Metrics/LineLength:
+Layout:
```

Comment:
Hi Joey.

Do you think we could explicitly set the name of the cop https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/LineLength ?

Reply:
I can do that. Realizing now the error message `Metrics/LineLength has the wrong namespace - should be Layout` was referring `Metrics` as the namespace instead of the entire thing